SpatialGDK/Documentation/known-issues.md

```diff
@@ -9,13 +9,9 @@ Known issue = any major user-facing bug or lack of user-facing feature that:
 | Issue | Date added | Ticket | Workaround? |
 |-------|-------------------|--------|-------------|
 | Order of reliable RPCs is not respected in case of unresolved UObject* parameter. | 2018-06-14 | [UNR-363](https://improbableio.atlassian.net/browse/UNR-336) | Potential user-code workaround to explicitly enforce the order of RPCs. |
-| Project build `PCHUsageMode` must be set to `UseExplicitOrSharedPCHs`. | 2018-06-27 |[UNR-354](https://improbableio.atlassian.net/browse/UNR-354)| |
-| Adaptive Unity builds are not supported. | 2018-07-28 | | Disable `bUseAdaptiveUnityBuild` following [Epic’s documentation](https://docs.unrealengine.com/en-US/Programming/UnrealBuildSystem/Configuration). |
 | Dynamic component replication. | 2018-07-03 | [UNR-366](https://improbableio.atlassian.net/browse/UNR-366) | Mimic behavior with static components. |
 | Components with replicated sub-objects fail. | 2018-07-03 |[UNR-417](https://improbableio.atlassian.net/browse/UNR-417) | Make all components top level. |
 | You can't have multiple replicated components of the same type on the same Actor. | 2018-07-03 | [UNR-416](https://improbableio.atlassian.net/browse/UNR-416) | None |
-| User code must be IWYU-compliant, meaning you must import all the headers you use.| 2018-07-13 |[UNR-380](https://improbableio.atlassian.net/browse/UNR-380) | None |
 | Schema generator fails when the destination file is locked. The workflow is less than ideal as you can't run it again until you restart the Unreal Editor. | 2018-07-16 | [UNR-350](https://improbableio.atlassian.net/browse/UNR-350) | Ensure destination files/folders are unlocked. |
 | Deleting entities directly from the Inspector causes undefined behavior. | 2018-07-17 | [UNR-425](https://improbableio.atlassian.net/projects/UNR/issues/UNR-425) | None |
 | We don't support listen servers. | 2018-07-30 | | Use dedicated servers instead. |
```

Comment:
```suggestion
| We don't support listen servers. | 2018-07-30 | | Use dedicated servers instead. |
| Server travel does not work in PIE. | 2018-07-30 | | Server travel work must be done with external or managed workers |
```
Long story short, since we only support `dedicated server` we can't use server travel in PIE since this requires multiple processes and this is not supported for dedicated servers...
Additionally seamless server travel is not supported in PIE in default Unreal anyway.